data/iamr-kumar.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "Ritik Kumar",
+  "type": "personal",
+  "bio": "Full Stack Engineer | Freelance Developer | Part-time writer",
+  "avatar": "https://github.com/iamr-kumar.png",
+  "tags": ["Javascript", "Typescript", "React", "NextJS", "Node", "Flutter"],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/itsr_kumar"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/iamr-kumar"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/ritikkumar29/"
+    }
+  ],
+  "links": [
+    {
+      "name": "Porfolio",
+      "url": "http://devritik.com",
+      "icon": "FaLink"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/itsr_kumar",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+
+    {
+      "name": "GitHub",
+      "url": "https://github.com/iamr-kumar",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/ritikkumar29/",
+      "icon": "FaLinkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "First Hackathon",
+      "date": "October 2020",
+      "icon": "FaCode",
+      "color": "blue",
+      "description": "Participated in my first hackathon and won it to got an internship offer",
+      "url": ""
```

Comment:
I don't think you should specify an URL if there's none. AFAIK, it could lead to an unwanted page.

Reply:
Oh I though the url field is required but can leave empty. Will fix it.